### PR TITLE
Cockpit 195

### PR DESCRIPTION
**Pull request by allisonkarlitskaya**
https://allisonkarlitskaya.github.io/cockpit-project.github.io/blog/cockpit-195.html

- Simplified support for external TLS termination
- Firewall: more support for zones
- Machines: additional storage options
- Machines: support for changing memory allocation
- Storage: select encryption type separately from filesystem

### CODE REVIEW

File: _posts/2019-05-29-cockpit-195.md [hunk 1]
```diff
@@ -0,0 +1,78 @@
...
+If the proxy runs on the same machine, then `cockpit-ws` can be run with
+the new `--for-tls-proxy` option, which will adjust the allowed
+[Origins](https://developer.mozilla.org/en-US/docs/Web/HTTP/Headers/Origin)
+and
+[Content-Security-Policy](https://developer.mozilla.org/en-US/docs/Web/HTTP/CSP)
+to http:// URLs. With this option, it's no longer necessary to
+explicitly configure
+[cockpit.conf](https://cockpit-project.org/guide/latest/cockpit.conf.5.html).
+
+### Firewall: more support for zones
```
Comment (Gundersanne) [inline _posts/2019-05-29-cockpit-195.md]: Generally the first word after the colon gets capitalized on our blog posts.

File: _posts/2019-05-29-cockpit-195.md [hunk 2]
```diff
@@ -0,0 +1,78 @@
...
+
+The machines page also gained support for changing the current and
+maximum memory allocations of a virtual machine.
+
+![Adjust virtual machine memory allocation](/images/machines-memory-allocation.png)
+
+### Storage: select encryption type separately from filesystem
+
+The selection of the filesystem type and filesystem encryption options
+have been uncoupled on the storage page, allowing for use of LUKS with
```
Comment (Gundersanne) [inline _posts/2019-05-29-cockpit-195.md]: Missing article? allowing for "the" use? Not sure here

File: _posts/2019-05-29-cockpit-195.md [hunk 3]
```diff
@@ -0,0 +1,78 @@
...
+
+![New encryption option](/images/storage-separate-encryption-option.png)
+
+### Try it out
+
+Cockpit 195 is available now:
+
+ * [For your Linux system](https://cockpit-project.org/running.html)
+ * [Source Tarball](https://github.com/cockpit-project/cockpit/releases/tag/195)
+ * [Fedora 29](https://bodhi.fedoraproject.org/updates/cockpit-195-1.fc29)
```
Comment (martinpitt) [inline _posts/2019-05-29-cockpit-195.md]: We just [stopped doing that](https://github.com/cockpit-project/cockpit/pull/11951), so please drop that line.

File: _posts/2019-05-29-cockpit-195.md [hunk 4]
```diff
@@ -0,0 +1,78 @@
...
+
+It's now easier to configure Cockpit's web server `cockpit-ws` to run
+behind a
+[TLS termination proxy](https://en.wikipedia.org/wiki/TLS_termination_proxy).
+If the proxy runs on the same machine, then `cockpit-ws` can be run with
+the new `--for-tls-proxy` option, which will adjust the allowed
+[Origins](https://developer.mozilla.org/en-US/docs/Web/HTTP/Headers/Origin)
+and
+[Content-Security-Policy](https://developer.mozilla.org/en-US/docs/Web/HTTP/CSP)
+to http:// URLs. With this option, it's no longer necessary to
```
Comment (martinpitt) [inline _posts/2019-05-29-cockpit-195.md]: Ah, you caught an earlier version of my PR -- I fixed that typo a minute after I told you on IRC. This should be "https://".

File: _posts/2019-05-29-cockpit-195.md [hunk 5]
```diff
@@ -0,0 +1,78 @@
...
+explicitly configure
+[cockpit.conf](https://cockpit-project.org/guide/latest/cockpit.conf.5.html).
+
+### Firewall: More support for zones
+
+In addition to the support for adding services to specific zones in the
+last release, the firewall page now allows the adding and removing of
```
Comment (martinpitt) [inline _posts/2019-05-29-cockpit-195.md]: "the" adding.. sounds a bit strange to me (but IANAENS)

File: _posts/2019-05-29-cockpit-195.md [hunk 6]
```diff
@@ -0,0 +1,78 @@
...
+![Create storage pool on physical disk device](/images/machines-new-pool-physical-disk-device.png)
+
+Finally, it is now possible to configure the disk cache policy when
+adding storage devices to virtual machines.
+
+![Adjust virtual machine disk cache policy](/images/machines-disk-cache-policy.png)
+
+### Machines: Support for changing memory allocation
+
+The machines page also gained support for changing the current and
```
Comment (martinpitt) [inline _posts/2019-05-29-cockpit-195.md]: "Machines" for consistency and correspondence to the menu entry.

File: _posts/2019-05-29-cockpit-195.md (hunk 5 above)
Comment (allisonkarlitskaya) [reply _posts/2019-05-29-cockpit-195.md]: As a native speaker, I'd say "IANANES" :joy:

File: _posts/2019-05-29-cockpit-195.md [hunk 7]
```diff
@@ -0,0 +1,75 @@
...
+
+![Adjust virtual machine memory allocation](/images/machines-memory-allocation.png)
+
+### Storage: Select encryption type separately from filesystem
+
+The selection of the filesystem type and filesystem encryption options
+have been uncoupled on the storage page, allowing for the use of LUKS
+with filesytems other than ext4 and xfs.
+
+![New encryption option](/images/storage-separate-encryption-option.png)
```
Comment (mvollmer) [inline _posts/2019-05-29-cockpit-195.md]: This is the most current screenshot: https://user-images.githubusercontent.com/3228183/57758953-84e84d00-7701-11e9-8fc6-2bccf46f4718.png

It has a gray box to group the encryption options.

Comment (allisonkarlitskaya) [reply _posts/2019-05-29-cockpit-195.md]: thx,

**Pull request merged**